less/forum/extension.less

```diff
@@ -1,6 +1,8 @@
 .Post--unapproved {
-  .Post-header,
   .Post-body,
+  .PostUser-avatar,
+  .PostUser > h3 > a,
```

Comment:
Not sure about this very specific selector, to be honest. Not having looked at the markup... is there a class name we can use instead?